lib/nandi/config.rb

```diff
@@ -56,13 +61,13 @@ class Config
     # @return [Integer]
     attr_accessor :concurrent_lock_timeout_limit
 
-    # The directory for Nandi migrations. Default: `db/safe_migrations`
+    # The directory for Nandi migrations. Default: `db/safe_migrations`.
     # @return [String]
-    attr_accessor :migration_directory
+    attr_writer :migration_directory
```

Comment:
Should we update all the attr_writers to do something like:
```ruby
def migration_directory=(value)
  @databases.default.migration_directory = value
end
```

Effectively we can refactor all of this config to multi-db, but only allow one db for now